orttraining/orttraining/test/python/_test_helpers.py

```diff
@@ -213,23 +221,29 @@ def run_with_pytorch_on_device(device, model, input_list, label_input, is_eval_m
     return forward_outputs, grad_outputs
 
 def run_with_ort_on_device(device, model, input_list, label_input, is_eval_mode=False):
-    model = copy.deepcopy(model)
-    model.to(device)
+    with torch.no_grad():
```

Comment:
nit
```
with torch.no_grad():
 process_1(...)
with torch.grad():
 do(...)
with torhc.no_grad().
```

Reply:
i will leave this in subquent PR changes, to avoid an extra approval needed. :) thanks